patch-package
--- a/node_modules/node-plop/lib/generator-runner.js
+++ b/node_modules/node-plop/lib/generator-runner.js
@@ -111,7 +111,7 @@ exports.default = function (plopfileApi, flags) {
 		// show the resolved value in the console
 		result => ({
 			type: cfg.type || 'function',
-			path: _colors2.default.blue(result.toString())
+			path: _colors2.default.blue(result && result.toString())
 		}),
 		// a rejected promise is treated as a failure
 		function (err) {
